scripts/ec2/tags.coffee

```diff
@@ -16,5 +16,13 @@ tags =
   removeSchedule: (msg, instances) ->
     return tags.addSchedule(msg, instances, "")
 
+  addReservation:(msg, instance, content) ->
+    ec2.createTags {Resources: [instance], Tags: content}, (err, res) ->
+      if err
+        console.log "Error creating tags: #{err}"
+        if msg
+          return msg.send "Error creating tags: #{err}" if err
```

Comment:
The nesting of these three `if` statements is confusing. It transpiles to:

```
if (err) {
  console.log("Error creating tags: " + err);
  if (msg) {
    if (err) {
      return msg.send("Error creating tags: " + err);
    }
  }
}
```

Is this intentional?

Reply:
```
if err
  console.log "Error creating tags: #{err}"
  return msg.send "Error creating tags: #{err}" if msg
```

might make more sense?

Reply:
No, we'll fix it.